docs/pages/branding/about.tsx

```diff
@@ -18,6 +18,139 @@ import BrandingRoot from 'docs/src/modules/branding/BrandingRoot';
+            guidelines-specific implementation,{' '}
+            <b>
+              we want Material-UI to be whatever is generally useful for application development
+            </b>
+            , all in the spirit of the Material Design guidelines.
+          </Typography>
+          <Typography>
+            Therefore, Material-UI does not only implement the Material Design guidelines, but is
+            also a general use UI library of components. It also means we offer components or
+            combinations that are simply not addressed in the design guidelines.
+          </Typography>
+        </Grid>
+      </Grid>
+      <Box sx={{ my: 15 }}>
+        <Grid container alignItems="center" spacing={4}>
+          <Grid item xs={12} sm={6} sx={{ order: [2, 1] }}>
+            <Typography>
+              <b>
+                We focus on providing all the low-level tools needed to build a rich user-interface
+                with React.
+              </b>{' '}
+              We implement the Material Design specification (which is a bar set quite high), you
+              are able to take advantage of it for your own business with any style customization
+              needed.
+            </Typography>
+            <Typography>
+              We want to see companies succeeding using Material-UI in a way that matches their
+              brand, close to the Material Design philosophy or not. We don&apos;t want them to feel
+              that their UI simply looks like another Google product.
+            </Typography>
+          </Grid>
+          <Grid item xs={12} sm={6} sx={{ order: [1, 2] }}>
+            <Box
+              sx={{
+                position: 'relative',
+                ml: 'auto',
+                maxWidth: 470,
+                '& div': {
+                  paddingTop: `${Math.round((940 / 1140) * 100)}%`,
+                },
+                '& img': {
+                  position: 'absolute',
+                  top: 0,
+                  left: 0,
+                  width: '100%',
+                  height: '100%',
+                  objectFit: 'cover',
+                },
+              }}
+            >
+              <div />
+              <img alt="" src="/static/branding/about/focus.jpg" loading="lazy" />
+            </Box>
+          </Grid>
+        </Grid>
+      </Box>
+      <Grid container alignItems="center" spacing={4}>
+        <Grid item xs={12} md={6}>
+          <BrandingQuote
+            author={{
+              avatar: '/static/branding/about/olivier.jpg',
+              name: 'Olivier Tassinari',
+              title: 'Co-founder',
+            }}
+            sx={{ width: '100%', maxWidth: { md: 470 } }}
+          >
+            We believe that React is set on a course to dominate the way UIs are built on the web
+            for a very long time. We also believe that the next major iteration is coming with
+            low-code. Material-UI is at the forefront of this transformation.
```

Comment:
Premature.

Reply:
```suggestion
            low-code. Material-UI aims to accelerate this transformation.
```

Reply:
I meant for any kind of public pronouncement.

Reply:
Low code means writing less code, one could argue that `@material-ui/core` is low code.